pkg/agent/agent.go

```diff
@@ -72,6 +72,14 @@ type agentData struct {
 
 var agent = agentData{}
 
+// getAgentTypes - return the agent types
```

Comment:
This is just returning a value, can't it be a variable?  Also, can you use the AgentType enum rather then the int representation?

Reply:
with no const for a map, I thought returning the map would be safer in the SDK; I am happy to change it.

Reply:
Its a private map, no?

Reply:
done